--- ./CcspCommonLibrary/source/util_api/ccsp_msg_bus/ccsp_base_api.c
+++ ./CcspCommonLibrary/source/util_api/ccsp_msg_bus/ccsp_base_api.c	2015-04-22 14:22:31.162821009 -0600
@@ -2269,8 +2269,10 @@
 }
 
 BOOLEAN waitConditionReady(void* hMBusHandle, const char* dst_component_id, char* dbus_path, char *src_component_id){
-    #define MAX_WAIT_TIME 10
-    #define TIME_INTERVAL 1000 // 1s
+    //#define MAX_WAIT_TIME 10
+    //#define TIME_INTERVAL 1000 // 1s
+    #define MAX_WAIT_TIME 5
+    #define TIME_INTERVAL 200 //
     #define CCSP_COMMON_COMPONENT_HEALTH_Green 3
     int times = 0;
     int ret   = 0;
